src/sequential_algorithms.md

```diff
@@ -0,0 +1,143 @@
+## Expressing Sequential Algorithms in Alan
```

Comment:
Thinking about it. Should this simply be the `@std/seq` documentation in the standard library that we haven't written yet?